client/src/js/components/map.jsx

```diff
@@ -39,7 +39,17 @@ var Map = React.createClass({
       }
     ).addTo(map);
 
-    L.geoJson(items).addTo(map);
+    var marker = new 
+    L.Icon({
+      iconUrl: "images/marker-icon.png"
+    });
+
+    function content (feature,layer) {
+      layer.bindPopup("<h1>" + feature.properties.Title + "</h1>");
+      layer.setIcon(marker);
+    };
+
+    L.geoJson(items, {onEachFeature: content}).addTo(map);
```

Comment:
I like that you created a separate function rather than using an anonymous function